src/main/java/Event.java

```diff
@@ -0,0 +1,13 @@
+public class Event extends Task {
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
```

Comment:
The optional @OverRide makes the code very clear and the readers are able to know you have updated changes to the functions from parent class.